kong/resty/dns/client.lua

```diff
@@ -1550,16 +1528,12 @@ local function connect(sock, host, port, sock_opts)
 
   if not targetIp then
     return nil, tostring(targetPort) .. ". Tried: " .. tostring(tryList)
-  else
-    -- need to do the extra check here: https://github.com/openresty/lua-nginx-module/issues/860
```

Comment:
Is this comment useless now?

Reply:
yes, this issue(in openresty) has been fixed now, so we dont care it. But at that time, dns client had to be compatible with the openresty issue and leave a comment